src/liger_kernel/transformers/monkey_patch.py

```diff
@@ -143,6 +145,8 @@ def apply_liger_kernel_to_gemma(
         modeling_gemma.CrossEntropyLoss = LigerCrossEntropyLoss
     if geglu:
         modeling_gemma.GemmaMLP = LigerGEGLUMLP
+    if fused_linear_cross_entropy:
```

Comment:
could you help to add an assertion here to make sure `fused_linear_cross_entropy` and `cross_entropy` are not True together, just like in llama monkey patch function? (looks like we dropped this in qwen2 too, would be helpful if you could add it for qwen too!)